src/command/setup.rs

```diff
@@ -51,7 +51,7 @@ mod os {
     ];
 
     pub fn setup_environment() -> Fallible<()> {
-        let user_home_dir = dirs::home_dir().ok_or(ErrorDetails::NoHomeEnvironmentVar)?;
+        let default_home_dir = dirs::home_dir().ok_or(ErrorDetails::NoHomeEnvironmentVar)?;
```

Comment:
The `user` references in this file actually refer to user settings on the system, not our previous concept of the "User platform", so I think these should remain (`user_home_dir`, `ReadUserPathError`, `WriteUserPathError`, etc.)